workloads/api-load/env.sh

```diff
@@ -4,6 +4,11 @@ TEST_CLEANUP=${TEST_CLEANUP:-true}
 export ES_SERVER=${ES_SERVER:-https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443}
 export ES_INDEX=ripsaw-api-load
```

Comment:
I know this is not part of the PR, but are we still usign the same index for `ripsaw` and for our results?

Reply:
I think we agreed for ocm-request-test. If you confirm, I will update this here.

Reply:
Thanks, I will push a separate PR for that.